### DISCUSSION TITLE
Bi-weekly evaluations

### DISCUSSION THREAD

**Discussion by shu-bamma** [larics/icuas22_competition#29] [Q&A]
@RobertMilijas
1. Do the bi-weekly evaluations starting from 15 March mean an evaluation per 2 weeks or 2 evaluations per week?
2. Can a team submit multiple runs in the bi-weekly period? if yes then what is the limit
3. What can we expect from the evaluations? just a breakdown of the scores or something like the logs/video of our drone?

**Comment by fpetric** [comment]
Hi @shu-bamma 

thanks for reaching out. We are still working on automating the evaluation process but here is the most recent information:
>
>     1. Do the bi-weekly evaluations starting from 15 March mean an evaluation per 2 weeks or 2 evaluations per week?
>
We will try to make evaluations more frequent. In the worst case that we envision, it would be 1 evaluation per two weeks (if we do everything manually for all teams). However, the plan is to make it more frequent (we aim for daily evaluation). 

>
>     2. Can a team submit multiple runs in the bi-weekly period? if yes then what is the limit
>
In case of longer periods between evaluations, we will try to evaluate a few solutions for each team (in that case, yes, you will be allowed multiple submissions). If we manage to achieve quicker turnaround on evaluations, only the most recent solution will be evaluated (even if you submit more than one, only the most recent will be evaluated.
 
>
>     3. What can we expect from the evaluations? just a breakdown of the scores or something like the logs/video of our drone?
>
Yes, we plan to release a table of scores on the web site of the competition after each evaluation run. The scores will contain the point total, points for each category and eventual penalty points. We haven't planned on releasing logs, but can consider it based on demand, but videos will not be feasible due to number of teams.

**Reply by luccagandra** [reply]
Hi @fpetric 
Our team has not submitted our solution yet. However, we want to know the frequency that the evaluation process is happening, but we are not finding the table of scores on the web site of the competition. Did you guys abandoned that idea?

**Reply by fpetric** [reply]
hi @luccagandra
we haven't abandoned the idea, it's just that we haven't had any successful runs yet :) we are trying to evaluate solutions once a day. Until we get a few successful runs in, we will probably just send emails to teams with their scores. But once things heat up, we will have results on the web
